### PR TITLE
🐞 fix(matchingnet): module filename

### PR DESCRIPTION
**Pull request by chenjiayun212**
core/model/meta/__init__.py imports a class from local module .matchingnet, but the filename of the module is not consistent with the code

### CODE REVIEW

File: core/model/metric/COSOC.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+                        temp[q,w,e,:,:] = sim[q,w,e,torch.arange(sim.size(3))!=idx_1[q,w,e,idx_2[q,w,e]].item(),torch.arange(sim.size(4))!=idx_2[q,w,e].item()]
+            sim = temp
+        sim = sim.reshape(b, M, N)
+        combination.append(F.relu(sim)*count)
+        combination = torch.stack(combination, dim = -1).sum(-1)
+            
+
+        classification_scores = torch.transpose(combination, 1,2)
+        return classification_scores
+
```
Comment (yuchen2003) [inline core/model/metric/COSOC.py:200]: asdf
